### PR TITLE
.env file default values removed

### PR DESCRIPTION
**Pull request by VirenderNayal**
Changes : 

- [x] All default values in .env file has been removed.
- [x] README.md is updated to guide to modify the .env file values.

Please ask if any changes are required.

Thank You.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,9 +92,31 @@ docker-compose --help
    cd MLOps
    ```
    
-The server can be configured by modifying the environment file found at ```/mlflow_server/.env```. The environment variable provided are given as an example, and should not be used for a production deployment.
+2. The server should be configured by modifying the environment file found at ```/mlflow_server/.env```. The environment variable shown are given as an example, and should not be used for a production deployment.
 
-2. Navigate to the cloned code repository and start the server. Any docker images that are not present on your local system will be pulled from dockerhub (which might take a while).
+They are compulsory to be set otherwise the server will crash.
```
Comment (laurencejackson) [inline README.md:97]: Please replace with:
"Setting these variables is a requirement, the server will fail to start if they are undefined."

**Review by laurencejackson — COMMENTED** [review]
Many thanks Virender, this is some great work. Happy to merge if you could address the minor wording change in the README.

**Review by laurencejackson — APPROVED** [review]
Perfect, I'll merge this in now, thanks for your contribution.

**Comment by VirenderNayal** [comment]
@laurencejackson Updated as required.

Thanks for you patience.

**Pull request merged**